src/physics/pbl_simple.f90

```diff
@@ -113,6 +113,10 @@ subroutine simple_pbl(th, qv, cloud, ice, qrain, qsnow, um, vm, pii, rho, z, dz,
                 ! diffusion for scalars
                 Kq_m(its:ite,k,j) = K_m(its:ite,k,j) / prandtl_m(its:ite,k,j)
 
+                ! rescale diffusion to cut down on excessive mixing
+                Kq_m(its:ite,k,j) = Kq_m(its:ite, k,j) / diffusion_reduction
+                Kq_m(its:ite,k,j) = Kq_m(its:ite, k,j) * dt / ((dz(its:ite,k,j) + dz(its:ite,k+1,j))/2)
```

Comment:
On Line 90 above, kte is set to a maximum of kme-1, so kte+1 should never be greater than kme. I believe I did this because I was getting some model-top error when not processing the top layer as well, but your comment says that we should be able to ignore this. Maybe we should discuss?